exercises/concept/annalyns-infiltration/.meta/exemplar.go

```diff
@@ -1,4 +1,4 @@
-package booleans
+package annalyns_infiltration
```

Comment:
Is `lower_snake_case` the correct naming convention here? In the repository, I saw `package thefar` (no spacing at all) but also `strings_package`, so I'm not sure which one I should use.

Reply:
Go uses a lot of shorthand naming, to comply with their [package naming conventions](https://blog.golang.org/package-names). I would just remove the underscore, or the more conventional approach would probably be `annalyn`.